charts/cluster-autoscaler/Chart.yaml

```diff
@@ -11,4 +11,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.37.0
+version: 9.37.1
```

Comment:
We'll need to bump this again as we merged a PR earlier which took us to 9.38.

I'd also want this to be a minor version bump given we're adding functionality.

Reply:
Done

Reply:
Done

Reply:
Done